tests/ha/test_control_pod_restart.py

```diff
@@ -383,3 +401,286 @@ def test_failover_control_pod_kubectl(self):
 
         LOGGER.info("ENDED: Verify IOs before and after control pod fails over, verify control "
                     "pod failover. (using kubectl command)")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40370")
+    @CTFailOn(error_handler)
+    def test_rd_wr_del_during_ctrl_pod_failover(self):
+        """
+        Verify READs, WRITEs and DELETEs during control pod failover.
+        """
+        LOGGER.info("STARTED: Verify READs, WRITEs and DELETEs during control pod failover.")
+
+        event = threading.Event()  # Event to be used to send when data pods going down
+        wr_bucket = HA_CFG["s3_bucket_data"]["no_bck_background_deletes"]
+        LOGGER.info("Step 1: Perform WRITEs with variable object sizes on %s buckets "
+                    "for parallel DELETEs.", wr_bucket)
+        wr_output = Queue()
+        del_output = Queue()
+        remaining_bkt = 10
+        del_bucket = wr_bucket - remaining_bkt
```

Comment:
do we need more number of buckets to be deleted in case failover takes time?

Reply:
For now I am creating 300 buckets and 290 we will be deleting, which is more than sufficient I feel. Because as per current architecture we will have to failover only single pod.